lib/screens/explore_screens/regions_screens/edit_region/interactor/viewmodel/edit_region_bloc.dart

```diff
@@ -0,0 +1,29 @@
+import 'dart:async';
+import 'package:bloc/bloc.dart';
+import 'package:equatable/equatable.dart';
+import 'package:seeds/datasource/remote/model/region_model.dart';
+
+part 'edit_region_events.dart';
+
+part 'edit_region_state.dart';
+
+class EditRegionBloc extends Bloc<EditRegionEvent, EditRegionState> {
+  EditRegionBloc(RegionModel region) : super(EditRegionState.initial(region)) {
+    on<OnRegionDescriptionChange>(_onOnRegionDescriptionChange);
+    on<OnEditRegionSaveChangesTapped>(_onEditRegionSaveChangesTapped);
+    on<ClearEditRegionPageCommand>((_, emit) => emit(state.copyWith()));
+  }
+
+  void _onOnRegionDescriptionChange(OnRegionDescriptionChange event, Emitter<EditRegionState> emit) {
+    if (event.regionDescription.isEmpty) {
+      emit(state.copyWith(newRegionDescription: event.regionDescription));
+    } else {
+      emit(
+        state.copyWith(newRegionDescription: event.regionDescription),
+      );
+    }
```

Comment:
I don't see nay difference between these 2 emits they do the same.

Reply:
You are right, I had some logic before but then It was removed and forgot to change this. 👍  fixed